src/main/common/utils/numericUtils.ts

```diff
@@ -14,3 +14,8 @@ export function toNumberOrUndefined (value: any): number {
 
   return isNaN(numberVal) ? undefined : numberVal
 }
+
+export function containsAThousandSeparator (input: string): boolean {
+  const output: string[] = input.split(',')
+  return output[output.length - 1].length >= 3
```

Comment:
This only checks the last part of the number string. What about e.g. `1,00,00,000`? That's not a correctly formatted number but will return true.

Reply:
And for `1,000000000` it returns true as well. I think that if thousands separator is used then it consistent formatting in all places should be enforced.

Reply:
numerals handles 1,0000000, however if any comma is present in less than 3, it is wrong value